server/middleware/tchannel-client/helpers/make-channel.js

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2021 Uber Technologies Inc.
+//
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+const path = require('path');
+const isIPv4 = require('is-ipv4-node');
+const TChannelAsThrift = require('tchannel/as/thrift');
+const { PEERS } = require('../constants');
+const lookupAsync = require('./lookup-async');
+
+const makeChannel = async client => {
+  const ipPeers = await Promise.all(
+    PEERS.map(peer => {
+      const [host, port] = peer.split(':');
+
+      if (!isIPv4(host)) {
+        return lookupAsync(host).then(ip => [ip, port].join(':'));
+      } else {
+        return peer;
+      }
+    })
+  );
+
+  const cadenceChannel = client.makeSubChannel({
+    serviceName: 'cadence-frontend',
+    peers: ipPeers,
+    requestDefaults: {
+      hasNoParent: true,
+      headers: { as: 'raw', cn: 'cadence-web' },
+    },
+  });
+
+  const tchannelAsThrift = TChannelAsThrift({
+    channel: cadenceChannel,
+    entryPoint: path.join(__dirname, '../../../idl/cadence.thrift'),
```

Comment:
its because its path has now moved two levels deeper from where the code was located earlier.
`server/middleware/tchannel-client.js` =>  `server/middleware/tchannel-client/helpers/make-channel.js`